app/build.gradle

```diff
@@ -126,6 +126,7 @@ android {
             buildConfigField "String", "EVENTLOG_WIKI", "\"commonswiki\""
             buildConfigField "String", "SIGNUP_LANDING_URL", "\"https://commons.m.wikimedia.org/w/index.php?title=Special:CreateAccount&returnto=Main+Page&returntoquery=welcome%3Dyes\""
             buildConfigField "String", "SIGNUP_SUCCESS_REDIRECTION_URL", "\"https://commons.m.wikimedia.org/w/index.php?title=Main_Page&welcome=yes\""
+            buildConfigField "boolean", "LOGIN_CREDENTIALS" , "false"
```

Comment:
A new field is not needed. You can simply use `BuildConfig.DEBUG`

Reply:
Will incorporate the changes